deployments.json

```diff
@@ -1078,5 +1078,11 @@
     "name": "DreyerX Mainnet",
     "chainId": 23451,
     "url": "https://scan.dreyerx.com/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract"
+  },
+  {
+    "name": "Lisk",
+    "chainId": 1135,
+    "url": "https://blockscout.lisk.com/address/0xA9d71E1dd7ca26F26e656E66d6AA81ed7f745bf0",
+    "address": "0xA9d71E1dd7ca26F26e656E66d6AA81ed7f745bf0"
```

Comment:
Is Lisk EVM compatible? I typically only merge non-standard addresses when a chain cannot support deployment to the standard address of `0xcA11bde05977b3631167028862bE2a173976CA11`